docs/topics/pages.rst

```diff
@@ -440,26 +440,25 @@ This is because ``Page`` enforces ordering QuerySets by path. Instead you must a
 
     news_items = NewsItemPage.objects.live().order_by('-publication_date')
 
-Page custom managers
+Custom Page managers
 --------------------
 
 ``Page`` enforces its own 'objects' manager in its ``__init__`` method, so you cannot add a custom manager at the 'objects' attribute.
```

Comment:
I had missed updating some of that documentation, you're right. I've rewritten those lines to actually be relevant now.

The two bits of example code are useful, as they do two different things. Making a custom Manager is a different thing to making a custom QuerySet, although they look quite similar. You have to inherit from different parent classes, and construct the Manager instance in a different manner, so documenting both approaches is required in my opinion.

The code for filtering for future events is already in a `future()` method.